ContentModeratorSDK/ReviewClient.cs

```diff
@@ -71,7 +57,26 @@ public ReviewClient(string subscriptionKey, string apiRoot, string clientId, str
                         Constants.HttpMethod.GET, metaData)
                         .ConfigureAwait(false);
         }
+        public async Task<ReviewVideoFramesResponse> GetVideoFrames(string teamName, string reviewId, string startSeed, string noOfRecords)
+        {
+            List<KeyValue> metaData = new List<KeyValue>();
```

Comment:
Lets name this variable as queryParameters, otherwise its confusing with out data model

Reply:
That's true. I'll rename the old SDK methods as well.